generators/bdk/java/templates/build.gradle.ejs

```diff
@@ -3,7 +3,7 @@ plugins {
 <% if (framework == 'java') { %>
     id 'application'
 <% } %>
-<% if (framework == 'spring') { %>
+<% if (framework == 'spring' || application == 'ext-app') { %>
```

Comment:
Right now the generated build files come with empty, that would be nice if we could get rid of them (it was the case before)

Reply:
I am also trying to find way to remove these empty lines